src/cli.ts

```diff
@@ -37,6 +50,37 @@ const main = async (argv: string[]) => {
       });
     });
 
+  program
+    .command("create-tasks")
+    .description(
+      "create tasks for reviewing the pre-translations of this branch"
+    )
+    .addOption(
+      new BranchNameOption(
+        "the Crowdin branch for which to create tasks"
+      ).makeOptionMandatory(true)
```

Comment:
Doesn’t need to be mandatory this one. Using the current branch name by default https://github.com/mollie/crowdin-cli/blob/master/src/config.ts#L17-L21